include/clang/Tooling/CrossTranslationUnit.h

```diff
@@ -60,7 +60,8 @@ class CrossTranslationUnit {
   const FunctionDecl *getCrossTUDefinition(const FunctionDecl *FD,
                                            StringRef CrossTUDir,
                                            StringRef IndexName,
-                                           StringRef CompilationDatabase = "");
+                                           StringRef CompilationDatabase = "",
+                                           bool DisplayCtuProgress = false);
```

Comment:
I think you could get the option from the compiler invocation, so this function argument should be redundant.

Reply:
Thinking a bit more, it is better to keep this as an argument.